unattended-upgrade

```diff
@@ -1640,6 +1647,30 @@ def run(options,             # type: Options
     if should_stop():
         return UnattendedUpgradesResult(False)
 
+    # check to see if want to auto-upgrade the devel release
+    if apt_pkg.config.find("Unattended-Upgrade::DevRelease") == "auto":
+        try:
+            import distro_info
+            devel = (distro_info.DistroInfo(DISTRO_ID) .
```

Comment:
This needs more work, as there's some special subclasses with special handling for debian/ubuntu.

Reply:
Should we block this PR for this or will you just add a FIXME in the code and we go ahead with this one?